app/assets/stylesheets/variables.scss

```diff
@@ -21,7 +21,7 @@ $tan: #fdf9f3;
 $bold-blue: #0045ff;
 
 $max-width: 1250px;
-$header-height: 46px; // todo: should be calculated automatically based on content
+$header-height: 56px; // todo: should be calculated automatically based on content
```

Comment:
I'm actually not sure if this can be calculated since we still have to push rest of the page when header is sticky...